cross/detox/Makefile

```diff
@@ -0,0 +1,25 @@
+PKG_NAME = detox
+PKG_VERS = 1.2.0
+PKG_VERS_CUT = -1.2.0
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = ftp://ftp.heanet.ie/mirrors/download.sourceforge.net/pub/sourceforge/d/de/detox/detox/1.2.0
+PKG_DIR = $(PKG_NAME)$(PKG_VERS_CUT)
+
+#DEPENDS = cross/ncurses
```

Comment:
if this isn't needed just change it to `DEPENDS = ` instead of commenting it out